docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -144,34 +144,45 @@ all labeled and additional information will be included below.
 
 
 .. list-table:: Outputs from Pregnancy Decision Tree
-  :widths: 3 15 15
+  :widths: 3 15 15 15
   :header-rows: 1
 
   * - Input
     - Data Source 
-    - Notes
+    - Overall stratification
+    - Use case 
   * - ANC attendance
     - Decision tree point
-    - 
+    - Age, scenario 
+    - Decision tree value & export for V&V
   * - Ultrasound status (AI assisted, standard, none)
     - Decision tree point
-    - 
+    - Scenario
+    - Decision tree value & export for V&V
   * - Gestational age at birth
     - :ref:`Pregnancy model <other_models_pregnancy_closed_cohort_mncnh>`
-    - 
+    - Decision tree value & export for V&V 
+    - Scenario
   * - Gestational age stated
```

Comment:
I think we should rename this "Gestational age estimate" or "Estimated gestational age" to be more clear. But perhaps that's out of the scope of this PR since this term appears in other places as well.

Reply:
I'm happy to make this change and all the instances elsewhere on this page, in this PR! But lmk if you'd rather it be in a separate PR